components/Tabs/Tabs.js

```diff
@@ -2,50 +2,57 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+    this.element=element;
+    // console.log(this.element)
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data= element.dataset.tab;
+    // console.log(this.data)
     
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
+    this.itemElement= document.querySelector(`.tabs-item[data-tab='${this.data}']`);
+    // console.log(this.itemElement);
```

Comment:
Well done to console log everything to see what's actually happening through these assignments.